### ISSUE TITLE
Problems on loading imagenet weight

### ISSUE THREAD

**Issue by jwfanDL** [alibaba-mmai-research/TAdaConv#5]
Hi @huang-ziyuan , thank you for your helps these days. We met problems when loading imagenet weights.
We first set _CHECKPOINT_FILE_PATH:_  in .yaml file as empty. We find lots of mismatches of models when loading pretrained weight.  
We find  keys in checkpoint mismatch is empty. Therefore, we download the checkpoint and reset _CHECKPOINT_FILE_PATH:_  and we find similar result. We find the lots of weight in resnet block does not matched. 
We will appreciate if you can tell us whether it is correct.
![截屏2022-02-23 下午4 27 07](https://user-images.githubusercontent.com/49198978/155283886-06371406-6317-4de2-ab13-a7a4f4d83366.png)

**Comment by huang-ziyuan** [comment]
Hi, thanks for using the code. 
Most of this is normal. The `rf` function (the weights that contains `rf` in it) and the BatchNorm for `avgpool` branch (weights with `avgpool_bn` in it) are randomly initialized or initialized in the code, **without** ImageNet initialization. 

Except for the first four items. Those bn weights are not loaded because of a bug, which we have fixed with a commit just now. 

Nevertheless, we found not loading the first four items does not have a large affect on the final result.
